star-ros2/src/star_gateway_bridge/test/test_message_converter.cpp

```
@@ -4,147 +4,147 @@
+#include <geometry_msgs/msg/twist.hpp> // NOLINT(build/include_order)
+#include <gtest/gtest.h>               // NOLINT(build/include_order)
+#include <sensor_msgs/msg/battery_state.hpp>
 
-namespace star
-{
+namespace star {
 
 // Test fixture for MessageConverter tests
-class MessageConverterTest : public ::testing::Test
-{
+class MessageConverterTest : public ::testing::Test {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: converter_

(cppcoreguidelines-pro-type-member-init)